### PR TITLE
Add content; Switch to Bootstrap

### PR DESCRIPTION
**Pull request by danielabreo**
.gitignore
  Add .DS_Store for Mac development

/images
  Add new images for homepage

index.html
  Switch to using template from Bootstrap 4

script.js
  Remove fact function
  Add easter-egg image function

New sections:
Screen Shot 2020-06-01 at 11 31 39 AM
Screen Shot 2020-06-01 at 11 32 03 AM
Screen Shot 2020-06-01 at 11 32 12 AM

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,17 +12,8 @@
...
-
-  // Pick a random fact.
-  const fact = facts[Math.floor(Math.random() * facts.length)];
-
-  // Add it to the page.
-  const factContainer = document.getElementById('fact-container');
-  factContainer.innerText = fact;
+function changeImage() {
+  document.getElementById("switch").src = "/images/hi.jpg";
+  document.getElementById("switch").style = "width:50%; padding-top:3em; padding-bottom:3em";
```
Comment (brucezheng) [inline portfolio/src/main/webapp/script.js:17]: Could we add these styles to the #switch css or make a class for this?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,24 +1,273 @@
...
+        <div class="col-md-5 pb-5 justify-content-center">
+          <div class="card justify-content-center">
+            <div class="card-body">
+              <h5 class="card-title">Work Experience</h5>
+              <h6 class="card-subtitle mb-2 text-muted">Research/Internship</h6>
+              <p class="card-text">
+              <ul>
+                <li><a href="https://www.rice.edu">Rice University</a></li>
+                <li><a href="https://www.nasa.gov">NASA</a></li>
+                <li><span style="color: #4285F4">G</span><span style="color: #DB4437">o</span><span
```
Comment (brucezheng) [inline portfolio/src/main/webapp/index.html:178]: Nit: (i think this is multiple lines)

Do
<li>
  <span>....</span>
</li>

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,24 +1,273 @@
...
-        and I enjoy putting my software engineering skills to use in solving real-world problems.
-      </p>
-      <img style="width:100%" src="/images/nasa.jpg" alt="Photo of Dan in the Space Shuttle"></img> 
-      <p>Click here to get a random fact about me:</p>
-      <button onclick="addRandomFact()">Get a fact!</button>
-      <div id="fact-container"></div>
-    </div>
-  </body>
-</html>
+<!doctype html>
```
Comment (ccyang314) [inline portfolio/src/main/webapp/index.html:1]: nit: change `doctype` to uppercase [1]. 

[1] https://html.spec.whatwg.org/multipage/syntax.html#the-doctype

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (danielabreo) [reply portfolio/src/main/webapp/script.js]: Converted to a class

**Pull request merged**